packages/types/src/api/web/global/scim/users.ts

```diff
@@ -0,0 +1,83 @@
+export interface ScimUser {
+  schemas: ["urn:ietf:params:scim:schemas:core:2.0:User"]
+  id: string
+  externalId: string
+  meta: {
+    resourceType: "User"
+    created: string
+    lastModified: string
+  }
+  userName: string
+  name: {
+    formatted: string
+    familyName: string
+    givenName: string
+  }
+  active: boolean
+  emails: [
+    {
+      value: string
+      type: "work"
+      primary: true
+    }
+  ]
+}
+
+export interface ScimListResponse {
+  schemas: ["urn:ietf:params:scim:api:messages:2.0:ListResponse"]
+  totalResults: number
+  Resources: ScimUser[]
```

Comment:
Could we do away with `ScimUser` interface and use `ScimUserResponse` here instead?
Looks like they have the same properties. 

Alternatively could do 
```ts
interface ScimUserResponse extends ScimUser {}
```
but preference would be on using the single interface, I think it's fine to have the response type be re-used in another response. I actually think that makes a lot of sense so the contracts line up for the singular vs the `Resources`.

Reply:
Totally agree, I actually changed that in a further PR. Changing it here